pkgs/applications/networking/sniffers/qtwirediff/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, qtbase
+, qmake
+, qtwayland
+, wrapQtAppsHook
+}:
+
+
+stdenv.mkDerivation {
+  pname = "qtwirediff";
+  version = "unstable-2022-01-15";
+
+  src = fetchFromGitHub {
+    owner = "aaptel";
+    repo = "qtwirediff";
+    rev = "64e163e0bfaf1e6d33379de153af53820c146411";
+    hash = "sha256-tEDUEMRiYyCFJYmD6EB+2ZocCAprDrmmJLX0Aa76djU=";
+  };
+
+  nativeBuildInputs = [
+    qmake
+    wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    qtbase
+    qtwayland
+  ];
+
+  postInstall = ''
+    mkdir $out/bin
+    cp qtwirediff $out/bin
+  '';
```

Comment:
```suggestion
    runHook postInstall
  '';
```

Reply:
same as above is there a reason to call the hook explicitly?

Reply:
if you don't, `.overrideAttrs (old: { postInstall = "blablabla"; });` won't work because your hook is not run, it's bad practice